sopel-modules/rat-board.py

```diff
@@ -1194,7 +1194,10 @@ def cmd_assign(bot, trigger, rescue, *rats):
         # Check if id returned is an id, decide for unidentified rats or rats.
         # print("i is " + str(i))
         idstr = str(i['id'])
-        if idstr != '0' and idstr != 'None':
+        if rat.lower() == rescue.client.lower():  # sanity check
```

Comment:
I would use rescue.data.IRCNick instead. The client field is the in-game name, but the IRCNick is the converted "irc compatible" name.